### PR TITLE
Dropdown

### PR DESCRIPTION
**Pull request by davidalvarez93**
+Dropdown en el modal añadir y editar Vuelo

### CODE REVIEW

File: database/routes/journeys.routes.js [hunk 1]
```diff
@@ -84,4 +84,50 @@ router.delete('/Flgts/:id', async (req,res)=>{
     res.json({status:'Flight deleted'});
 });
 
+
+/////////////////////////////////
```
Comment (ezrra) [inline database/routes/journeys.routes.js:88]: Remove /////////...

File: database/routes/journeys.routes.js [hunk 2]
```diff
@@ -84,4 +84,50 @@ router.delete('/Flgts/:id', async (req,res)=>{
...
+    const ActualizarPermiso = {
+        P_Numero, P_UserName, 
+        P_Date, P_For_Airports, P_For_Flights, 
+        P_For_Usernames
+    };
+    await permissionsModel.findByIdAndUpdate(req.params.id, ActualizarPermiso);
+    res.json({status:'Permission updated'});
+});
+
+//////////////////////////////
```
Comment (ezrra) [inline database/routes/journeys.routes.js:131]: Remove /////...

File: src/components/AddAirport.js [hunk 3]
```diff
@@ -52,44 +52,52 @@ class Airport extends Component{
             [name]: value
         });
     }
-
+    componentWillUnmount(){
+        console.log("modal unmounted");
```
Comment (ezrra) [inline src/components/AddAirport.js:55]: Remove console.log

File: src/components/AddFlights.js [hunk 4]
```diff
@@ -1,24 +1,46 @@
 import React, {Component } from 'react';
+import {Dropdow, Button, NavItem ,Input} from 'react-materialize';
+
 import './estilos.css'
 
+
```
Comment (ezrra) [inline src/components/AddFlights.js]: Remove blank line unnecessary

File: src/components/AddUser.js [hunk 5]
```diff
@@ -6,21 +6,22 @@ class Permisos extends Component{
...
+            P_UserName:'',
+            P_Date:'',
+            P_For_Airports:'',
+            P_For_Flights:'',
+            P_For_Usernames :'',
+            _id:''
         };
         this.handleChange=this.handleChange.bind(this);
-        this.AddUsuario=this.AddUsuario.bind(this);
+        this.AddPermiso=this.AddPermiso.bind(this);
```
Comment (ezrra) [inline src/components/AddUser.js]: Rename variable and functions to english language

File: src/components/AddUser.js [hunk 6]
```diff
@@ -30,15 +31,16 @@ class Permisos extends Component{
...
-                Ap_Address:''
+                P_Numero:'',
+                P_UserName:'',
+                P_Date:'',
+                P_For_Airports:'',
+                P_For_Flights:'',
+                P_For_Usernames :''
             });
-            this.fetchAirports();
+            this.fetchPermisos();
```
Comment (ezrra) [inline src/components/AddUser.js:44]: Rename variable and functions to english language

File: src/components/AddUser.js [hunk 7]
```diff
@@ -56,35 +58,35 @@ class Permisos extends Component{
     render(){
         return(
             <div>
-            <form onSubmit={this.AddUsuario}>
+            <form onSubmit={this.AddPermiso}>
```
Comment (ezrra) [inline src/components/AddUser.js]: Rename variable and functions to english language

File: src/components/AddUser.js [hunk 8]
```diff
@@ -56,35 +58,35 @@ class Permisos extends Component{
...
                                         <div className="row">
                                             <div className="input-field col s10 push-s1 MiInputField">
-                                                <input name="UserName" value={this.state.UserName} onChange={this.handleChange} type="text" placeholder="UserName"/>
+                                                <input name="P_Numero" value={this.state.P_Numero} onChange={this.handleChange} type="text" placeholder="Número"/>
```
Comment (ezrra) [inline src/components/AddUser.js]: Rename variable and functions to english language

File: src/components/Airports.js [hunk 9]
```diff
@@ -49,114 +48,124 @@ class Airport extends Component{
...
     }
-    nolose(){
-        console.log("si funciona");
+
+    searchUpdated (term) {
+        this.setState({searchTerm: term})
+    }
+
+    componentWillUnmount(){
+        console.log("modal unmounted");
```
Comment (ezrra) [inline src/components/Airports.js:62]: Remove console.log

File: src/components/Flights.js [hunk 10]
```diff
@@ -45,7 +46,7 @@ class Airport extends Component{
             .then(data=>{
                 this.setState({Vuelos:data});
             });
-        }
+    }
 
     DeleteFlights(id){
         if(window.confirm('Estas seguro de eliminar el aeropuerto')){
```
Comment (ezrra) [inline src/components/Flights.js]: Add characters "Signo de interrogación" at question

File: src/components/EditFlights.js [hunk 11]
```diff
@@ -110,7 +144,16 @@ class Airport extends Component{
                 <div className="row"></div>
                 <div className="row">
                     <div className="col s1 push-s9">
-                        <button onClick={()=>{this.GuardarCambios()}}  className="btn light-blue darken-3 large">
+                        <button onClick={()=>{this.GuardarCambios()}}  className="btn light-blue darken-3 large" disabled={
+                            !this.state.Origen ||
```
Comment (ezrra) [inline src/components/EditFlights.js:174]: Rename variables Spanish to English language

File: src/components/Permisos.js [hunk 12]
```diff
@@ -1,46 +1,64 @@
...
+import { Modal, Button, Row, Col, Input} from 'react-materialize';
+import SearchInput, {createFilter} from 'react-search-input';
+import Switch from "react-switch";
 import AddUserView from './AddUser';
 
 import './estilos.css'
 
 
-const KEYS_TO_FILTERS = ['Ap_Name', 'Ap_Country', 'Ap_State']
+const KEYS_TO_FILTERS = ['P_Numero', 'P_UserName', 'P_Date']
```
Comment (ezrra) [inline src/components/Permisos.js:10]: Rename variables Spanish to English language

File: src/components/Permisos.js [hunk 13]
```diff
@@ -1,46 +1,64 @@
...
 class Permisos extends Component{
     constructor(props){
         super(props);
         this.state={
-            Ap_Name:'', 
-            Ap_Country:'',
-            Ap_State:'',
-            Ap_City:'',
-            Ap_Address:'',
+            P_Numero:'',
```
Comment (ezrra) [inline src/components/Permisos.js:17]: Rename variables Spanish to English language

File: src/components/Permisos.js [hunk 14]
```diff
@@ -1,46 +1,64 @@
...
+            P_UserName:'',
+            P_Date:'',
+            P_For_Airports:'',
+            P_For_Flights:'',
+            P_For_Usernames :'',
             _id:'',
-            Aeropuertos:[],
-            searchTerm: ''
+            Permisos:[],
```
Comment (ezrra) [inline src/components/Permisos.js]: Rename variables Spanish to English language

File: src/components/Permisos.js [hunk 15]
```diff
@@ -1,46 +1,64 @@
...
        
     }
 
-    fetchAirports(){
-        fetch('http://localhost:3001/api/journeys/Apts/')  
+    handleChange(checked) {
+        this.setState({ checked });
+      }
+
+    fetchPermisos(){
```
Comment (ezrra) [inline src/components/Permisos.js]: Rename variables Spanish to English language

File: src/components/Permisos.js [hunk 16]
```diff
@@ -1,46 +1,64 @@
...
+        const name = target.name;
+    
+        this.setState({
+          [name]: value
+        });
+      }
+
     componentDidMount(){
-        this.fetchAirports();
+        this.fetchPermisos();
```
Comment (ezrra) [inline src/components/Permisos.js]: Rename variables Spanish to English language

**Review by ezrra — CHANGES_REQUESTED** [review]
Remove the node_modules folder.
Rename variables Spanish to English language.

**Comment by davidalvarez93** [comment]
+Validación de campos 
-numero
-Fecha
-Hora
+Agregar Usuarios
+Ver Permisos
+Etiquetas de Formularios

**Pull request merged**